test/providers/cuda_helpers.cpp

```diff
@@ -287,6 +320,25 @@ CUcontext get_current_context() {
     return context;
 }
 
+CUresult set_context(CUcontext required_ctx, CUcontext *restore_ctx) {
```

Comment:
This could be `static`, it's not used outside of `cuda_helpers.cpp`.

Reply:
done